.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "src/c/nat/miniupnp"]
+	path = src/c/nat/miniupnp
+	url = https://github.com/miniupnp/miniupnp
```

Comment:
I think git failed to fetch, this lib

Reply:
- [ ] git submodule update --force --recursive --init --remote

Reply:
I think we can insert it into the Makefile